README.md

```diff
@@ -41,6 +41,23 @@ described earlier.
     cargo test
 
 
+Travis
+------
+CSearch is continuously integrated on [Travis CI].
+
+To update encrypted credentials in the `.travis.yml` file, use the
+[Travis command line tool].
+
+    gem install travis
```

Comment:
(If we want to do this inside the container though, I'm not sure what the current docker-in-docker story is like.)